<?xml version="1.0"?>
<launch>
  <arg name="press_dis" default="0.05"/> <!-- unit --> <!-- CHGH 4.1 -->
 
  <!-- arm press button -->
  <node pkg="elevator_caller3" type="scara_arm_press" name="scara_arm_press" output="screen">
    <param name="press_dis" value="$(arg press_dis)"/> 
  </node>

</launch>
